packages/react-components/src/index.ts

```diff
@@ -1,6 +1,5 @@
 // Utilities
 export {
-  RendererContext,
```

Comment:
These things are intentionally not re-exported, consumers should be `RendererProvider` to pass renderers.